src/Cake.Common/Tools/MSBuild/MSBuildRunner.cs

```diff
@@ -173,7 +173,23 @@ protected override IEnumerable<FilePath> GetAlternativeToolPaths(MSBuildSettings
                 throw new ArgumentNullException("settings");
             }
 
-            var path = MSBuildResolver.GetMSBuildPath(_fileSystem, _environment, settings.ToolVersion, settings.PlatformTarget ?? PlatformTarget.MSIL);
+            MSBuildTarget target = settings.MSBuildTarget ?? MSBuildTarget.Default;
```

Comment:
Since `MSBuildTarget` shouldn't be nullable, there is no need for null coalescing here.